### PR TITLE
Add comments for python version and image in python publish workflow

### PR DESCRIPTION
**Pull request by karenc**

### CODE REVIEW

**Comment by mmulich** [comment]
This gets us one step closer to the installation check. If the macOS build ever finishes we'll be good to go. I was unable to see any output from the job. It may be stuck.

Note, I've run into this in #58. One of the other things I mentioned in chat was that macOS is likely going to need `opencv-python-headless` as opposed to `opencv-python`.

File: .github/workflows/python-publish.yml [hunk 1]
```diff
@@ -26,7 +26,9 @@ jobs:
 
       - name: Build wheel
         env:
-          CIBW_SKIP: cp27-* pp27-* pp36-*
+          CIBW_BUILD: cp3?-*
+          CIBW_SKIP: cp35-* *-win32 *-win_amd64 *-manylinux_i686 *-manylinux_aarch64 *-manylinux_ppc64le *-manylinux_s390x
+          CIBW_MANYLINUX_X86_64_IMAGE: quay.io/pypa/manylinux2014_x86_64
```
Comment (mmulich) [inline .github/workflows/python-publish.yml:31]: Can you put a comment above this to say that this image was used to allow the testing installation to work?

File: .github/workflows/python-publish.yml [hunk 2]
```diff
@@ -26,7 +26,9 @@ jobs:
 
       - name: Build wheel
         env:
-          CIBW_SKIP: cp27-* pp27-* pp36-*
+          CIBW_BUILD: cp3?-*
+          CIBW_SKIP: cp35-* *-win32 *-win_amd64 *-manylinux_i686 *-manylinux_aarch64 *-manylinux_ppc64le *-manylinux_s390x
```
Comment (mmulich) [inline .github/workflows/python-publish.yml:30]: Can you put a comment on this to simply say it builds and tests for CPython >=3.6 on x86_64 architecture?

**Comment by karenc** [comment]
The commits have been merged as part of #69 but I'll add the comments

**Pull request merged**